packages/account/src/Components/personal-details/personal-details.jsx

```diff
@@ -75,7 +75,7 @@ const PersonalDetails = ({
         onCancel(current_step, goToPreviousStep);
     };
 
-    const is_qualified_for_idv = shouldShowIdentityInformation({
+    const is_for_idv = shouldShowIdentityInformation({
         account_status,
         account_settings,
```

Comment:
Can we get these data from `useGetAccountStatus` and `useSettings`. I think better to get these data from the server instead of going through stores 😢. Will greatly reduce props as well